subsys/logging/dictionary/Kconfig

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2021 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+config LOG_DICTIONARY_SUPPORT
```

Comment:
We already have a `Logging --> Output Formatting` menu here:
https://github.com/zephyrproject-rtos/zephyr/blob/master/subsys/logging/Kconfig.formatting

So please move this config in there, for example after the MIPI_SYST formatter.
https://github.com/zephyrproject-rtos/zephyr/blob/2cff32b6c4c68a9ed24375ee42b1d3aa79b7195d/subsys/logging/Kconfig.formatting#L24-L29